python/dask_cudf/dask_cudf/backends.py

```diff
@@ -15,6 +15,13 @@
     is_scalar,
 )
 
+try:
+    from dask.dataframe.methods import tolist_dispatch
+except ImportError:  # Fallback for Dask without `tolist` dispatch
+    from dask.utils import Dispatch
+
+    tolist_dispatch = Dispatch("tolist")
```

Comment:
What does this `Dispatch("tolist")` accomplish if the dask codebase isn't set up to use a dispatch on the `tolist` method?

Reply:
We are registering some functions below using `tolist_dispatch`. So this gives them some way to still do that (even though older versions of Dask won't use them).

Reply:
So this just solves the import error, but all of the implementations would still be broken.

Reply:
This would work but if someone has `cudf`-`nightly` + `dask`-`stable`. We will anyway see the errors from `.tolist` as upstream dask in older releases will directly call `Index.tolist` or `Series.tolist`.
`dask_cudf.sorting.sort_values` will fail too as we will not have `methods.tolist` function.